ws-nextjs-app/pages/[service]/send/[id]/FormField/Checkbox.tsx

```diff
@@ -22,9 +24,11 @@ export default ({
         type="checkbox"
         checked={value as boolean}
         onChange={e => handleChange(e.target.name, e.target.checked)}
-        aria-invalid={!isValid}
-        aria-required={required}
-        aria-describedby={describedBy}
+        {...(submitAttemptCount > 0 &&
```

Comment:
This bit of script is repeated quite a bit, but there is a variation to it in the Checkbox component.

Reply:
I think this is ok. Less 'DRY', but I think thats fine in this case as overtime the complexity of the forms may grow, so having this separation between the input types is good imo.